src/js/modules/wallet/controllers/receive/receive.ctrl.js

```diff
@@ -0,0 +1,179 @@
+(function () {
+    "use strict";
+
+    angular.module("blocktrail.wallet")
+        .controller("ReceiveCtrl", ReceiveCtrl);
+
+    function ReceiveCtrl($scope, $rootScope, $q, CONFIG, activeWallet, settingsService, CurrencyConverter, Currencies,  trackingService) {
+
+        var walletData = activeWallet.getReadOnlyWalletData();
+        var settingsData = settingsService.getReadOnlySettingsData();
+        // get current active wallets native currency
+        var nativeCurrency = CONFIG.NETWORKS[walletData.networkType].TICKER;
+
+        var listenerGroupValues;
+
+        $rootScope.pageTitle = 'RECEIVE';
+
+        $scope.isLoading = true;
+
+        $scope.address      = null;
+        $scope.path         = null;
+        $scope.bitcoinUri   = null;
+        $scope.qrcode       = null;
+
+        $scope.newRequest   = {
```

Comment:
now that we're refactoring lets stop doing this extra spaces to align the `=` signs

Reply:
plz also check rest of the file

Reply:
just search for `  =` in this PR and replace all of them ;)